orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/util/ContextParameterProcessor.groovy

```diff
@@ -351,7 +352,14 @@ abstract class ContextUtilities {
 
   static String fromUrl(String url) {
     URL u = contextFunctionConfiguration.get().urlRestrictions.validateURI(url).toURL()
-    return u.getText()
+    String response = ""
```

Comment:
we typically throw an exception when one of these methods fails which ultimately results in the expression remaining as is in the context

in this case I _think_ I'm ok with empty stringing out after we've retried sufficient times otherwise we may continue to try to hit that URL